drivers/wdm/audio/backpln/portcls/miniport_dmus.cpp

```diff
@@ -7,13 +7,8 @@
  *
  */
 
-#include "private.hpp"
-
-#ifndef YDEBUG
-#define NDEBUG
-#endif
 
-#include <debug.h>
+#include "private.hpp"
```

Comment:
strip one newline above #include "private.hpp" as it will help git compare to better align what should be aligned.

Reply:
Fixed.